packages/react-components/react-dialog/src/stories/DialogNested.stories.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+import { Dialog, DialogTrigger, DialogSurface, DialogTitle, DialogActions, DialogBody } from '@fluentui/react-dialog';
+import { Button } from '@fluentui/react-components';
+
+export const Nested = () => {
+  return (
+    <>
+      <Dialog>
+        <DialogTrigger>
+          <Button>Open dialog</Button>
+        </DialogTrigger>
+        <DialogSurface aria-label="label">
+          <DialogTitle>Dialog title</DialogTitle>
+          <DialogActions>
+            <Dialog>
+              <DialogTrigger>
+                <Button appearance="primary">Open inner dialog</Button>
+              </DialogTrigger>
+              <DialogSurface aria-label="label">
+                <DialogTitle>Inner dialog title</DialogTitle>
+                <DialogBody>⚠️ just because you can doesn't mean you should have nested dialogs ⚠️</DialogBody>
```

Comment:
Love this message 💖💖

Reply:
honestly an image would be far better

![image](https://user-images.githubusercontent.com/20744592/180748097-fca13cdb-f287-4d5b-9670-d421c2045a07.png)